gemrb/core/Scriptable/Door.cpp

```diff
@@ -324,13 +324,13 @@ void Highlightable::TryDisarm(Actor* actor)
 {
 	if (!Trapped || !TrapDetected) return;
 
-	int skill = actor->GetStat(IE_TRAPS);
+	int skill = actor->GetStat(IE_LOCKPICKING);
 	int roll = 0;
 	int bonus = 0;
 	int trapDC = TrapRemovalDiff;
 
 	if (core->HasFeature(GFFlags::RULES_3ED)) {
-		skill = actor->GetSkill(IE_TRAPS);
+		skill = actor->GetSkill(IE_LOCKPICKING);
```

Comment:
These two are outright wrong even by the in-game description.

Reply:
My mistake. I always thought that lockpicking was used there.